src/test/java/racingcar/model/CreateObjectCarsTest.java

```diff
@@ -0,0 +1,68 @@
+package racingcar.model;
+
+import org.assertj.core.api.Assertions;
+import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.api.Test;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+
+public class CreateObjectCarsTest {
+    @Test
+    void 차_이름_문자열을_배열로_잘_저장하는_지_확인() {
+        //Given
+        CreateObjectCars createObjectCars = new CreateObjectCars();
+        String carNamesStr = "CarA,CarB,CarC";
+
+        //When
+        String[] carNamesArr = createObjectCars.splitCarNames(carNamesStr);
+        SoftAssertions softly = new SoftAssertions();
+
+        //Then
+        softly.assertThat(carNamesArr.length).isEqualTo(3);
+        softly.assertThat(carNamesArr[0]).isEqualTo("CarA");
+        softly.assertThat(carNamesArr[1]).isEqualTo("CarB");
+        softly.assertThat(carNamesArr[2]).isEqualTo("CarC");
```

Comment:
```suggestion
SoftAssertions.assertSoftly(softly -> {
        softly.assertThat(carNamesArr.length).isEqualTo(3);
        softly.assertThat(carNamesArr[0]).isEqualTo("CarA");
        softly.assertThat(carNamesArr[1]).isEqualTo("CarB");
        softly.assertThat(carNamesArr[2]).isEqualTo("CarC");
});
```

저는 보통 위처럼 사용하는 편이네요
이러면 `SoftAssertions softly = new SoftAssertions();`를 따로 선언할 필요가 없다는 점이 있을 것 같아요

Reply:
수정했습니다!
저렇게 간결하게 적을 수 있는 지는 몰랐네요 ㅎㅎㅎ
애용하겠습니다!!

(피드백을 놓치지 않기 위해 댓글을 달아두는 것이니 따로 확인 안 하셔도 됩니다:))